src/sync/submitters/types.ts

```diff
@@ -103,6 +103,7 @@ export type DROPPED = 1;
 export type DEDUPED = 2;
 export type ImpressionDataType = QUEUED | DROPPED | DEDUPED
 export type EventDataType = QUEUED | DROPPED;
+export type UpdatesFromSSEEnum = 'Splits' | 'MySegments';
```

Comment:
you can reuse some constants here `export type UpdatesFromSSEEnum = SPLITS | MY_SEGMENT;`